#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_docs_tries.dpatch by Jonas Meurer <mejo@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: patch documentation about the --tries option

@DPATCH@

--- cryptsetup-1.0.4~rc2/man/cryptsetup.8
+++ cryptsetup-1.0.4~rc2/man/cryptsetup.8
@@ -58,7 +58,7 @@
 .IP
 add a new key file/passphrase. An existing passphrase or key file (via \-\-key-file) must be supplied. The key file with the new material is supplied as after \fIluksAddKey\fR as positional argument. <options> can be [\-\-key-file].
 .PP
-\fIluksDelKey\fR <key slot number>
+\fIluksDelKey\fR <device> <key slot number>
 .IP
 remove key from key slot. A remaining passphrase or key file (via \-\-key-file) must be supplied. <options> can be [\-\-key-file].
 .PP
@@ -91,7 +91,7 @@
 use file as key material. With LUKS, key material supplied in key files via \-d are always used for existing passphrases. If you want to set a new key via a key file, you have to use a positional arg to \fIluksFormat\fR or \fIluksAddKey\fR. If the key file is "-", stdin will be used.
 .TP
 .B "\-\-key-size, \-s"
-set key size in bits. Usually, this is 128, 192 or 256. Can be used for \fIcreate\fR or \fIluksFormat\fR, all other LUKS actions will ignore this flag, as the key-size is specified by the partition header. Default is 0.
+set key size in bits. Has to be a multiple of 8 bits. The key size is limited by the used cipher. See output of /proc/crypto for more information. Can be used for \fIcreate\fR or \fIluksFormat\fR, all other LUKS actions will ignore this flag, as the key-size is specified by the partition header. Default is 128.
 .TP
 .B "\-\-size, \-b"
 force the size of the underlaying device in sectors.
